treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -2,10 +2,10 @@
   pk: 1
   fields:
     framework: 1
-    keywords: perf, regression, talos-regression
-    default_component: Performance
+    keywords: perf, perf-alert, regression, talos-regression
+    default_component: General
```

Comment:
Is there a reason for changing the default component?

Reply:
I had a git problem. No, there is not a reason for changing the default component. I will solve this.